### PR TITLE
Updates for Bundler

### PR DESCRIPTION
**Pull request by chrisfinazzo**
Complement to #5168.

/cc @jekyll/documentation

### CODE REVIEW

File: site/_docs/installation.md [hunk 1]
```diff
@@ -31,16 +32,18 @@ earlier, for CoffeeScript support).
...
+## Install with Bundler and RubyGems
 
-The best way to install Jekyll is via
+The best way to install Jekyll is using a Gemfile via Bundler
 [RubyGems](http://rubygems.org/pages/download). At the terminal prompt,
 simply run the following command to install Jekyll:
 
 {% highlight shell %}
-$ gem install jekyll
+$ gem install bundler
```
Comment (chrisfinazzo) [inline site/_docs/installation.md]: Do we need to install both Jekyll and Bundler outside of the Gemfile for this to work?

Comment (envygeeks) [inline site/_docs/installation.md]: Only bundler need be installed outside the Gemfile.  Jekyll can stay within it, but we'll have to update and backtrack to make sure everything uses `bundle exec` that way we've consistent documentation.

**Comment by chrisfinazzo** [comment]
👌🏻

File: site/_docs/installation.md [hunk 2]
```diff
@@ -31,16 +32,18 @@ earlier, for CoffeeScript support).
...
 {% endhighlight %}
 
+Now you can install Jekyll simply by running `bundle install`.
```
Comment (parkr) [inline site/_docs/installation.md]: Where's your Gemfile being created? This only works with a Gemfile. The normal way without a Gemfile is still `gem install jekyll`

Comment (chrisfinazzo) [inline site/_docs/installation.md]: It was in the original PR (#4368) but later removed. If `new.rb` defines `gemfile_contents` and creates a scaffold, that's one less step we need to tell people about.

831b9a92-c0f5-11e5-8f9c-8ca6b686c438


@ashmaroli has done a fair bit of work on this in #5203 to extend the capabilities of `new` and flesh out the site template.

Might also be useful for themes 🤔

Comment (ashmaroli) [inline site/_docs/installation.md]: The Gemfile gets created with `gem "jekyll", "3.2.1"` included even with the latest release, v3.2.1. I'm just taking the generation outside `new.rb` to an erb template.

Comment (ashmaroli) [inline site/_docs/installation.md]: For the `jekyll new` command to run, jekyll has to be installed in the system.
So for first time users, `gem install jekyll` is the way.

bundle install will install the minima gem or any theme gem

File: site/_docs/installation.md [hunk 3]
```diff
@@ -31,16 +32,18 @@ earlier, for CoffeeScript support).
   </p>
 </div>
 
-## Install with RubyGems
+## Install with Bundler and RubyGems
 
-The best way to install Jekyll is via
+The best way to install Jekyll is using a Gemfile via Bundler
 [RubyGems](http://rubygems.org/pages/download). At the terminal prompt,
```
Comment (ashmaroli) [inline site/_docs/installation.md]: The standard way to install Jekyll on a new system is still `gem install jekyll`
But if for some reason, one decides to try Jekyll out using an existing repo, and if that repo was created with Jekyll 3.2, then, the best way for him to use bundler. He'll be able to install jekyll with all its dependencies and the repo's theme, with one command `bundle install`
To get the repo-site served just after installation with minimal time, from the repo root, he just needs to enter 
```
$ bundle && bundle exec jekyll s
```

**Comment by benbalter** [comment]
Just for completeness, an alternative proposal that came out of https://talk.jekyllrb.com/t/jekyll-3-2-what-are-the-benefits-for-users/2743/26 was to change the first line to `gem install jekyll minima` and skip Bundler entirely.

**Comment by ashmaroli** [comment]
@benbalter, `gem install jekyll minima` Cool.. Two gems with one command!
But let the bundler stay.. its easier to manage theme versions with it..

**Comment by chrisfinazzo** [comment]
Closing this - will reorganize and open a new PR to match up with the current state of things.

**Pull request closed without merging**